AttitudeManager/Inc/AM.hpp

```diff
@@ -27,6 +27,20 @@ typedef struct {
 
 class AttitudeManager {
    public:
+
+    //Constants used for mapping values
+    static const int PITCH_LIM_MAX = 30;
+    static const int PITCH_LIM_MIN = -30;
+
+    static const int ROLL_LIM_MAX = 40;
+    static const int ROLL_LIM_MIN = -40;
+
+    static const int YAW_LIM_MAX = 250;
+    static const int YAW_LIM_MIN = 0;
+
+    static const int INPUT_MAX = 100;
+    static const int INPUT_MIN = -100;
```

Comment:
See above comments, sorry about the confusion once again :''

Reply:
Removed the axis limits and added them config.hpp, left the input max/min here tho

Reply:
i think we can assume that input/output will always be scaled, but it's always good to do bounds checking!

Just make sure that you _actually_ do the `assert()` call within AM::run()